src/boot/replaceRevive.js

```diff
@@ -57,25 +57,25 @@ function replacer(key, value) {
 
   switch (Object.getPrototypeOf(origValue)) {
     // Flow bug: https://github.com/facebook/flow/issues/6110
-    case (ZulipVersion.prototype: $FlowFixMe):
+    case (ZulipVersion.prototype: $FlowIssue):
```

Comment:
nit in commit message:

    flow [nfc]: Better indicate flow bugs in a few suppressions.

Should read "Flow bugs".